src/datatype/operators_svecf32.rs

```diff
@@ -88,6 +88,47 @@ fn _vectors_svecf32_operator_minus(lhs: SVecf32Input<'_>, rhs: SVecf32Input<'_>)
     SVecf32Output::new(SVecf32Borrowed::new(lhs.dims(), &indexes, &values))
 }
 
+/// Calculate the element-wise multiplication of two sparse vectors.
+#[pgrx::pg_extern(immutable, strict, parallel_safe)]
+fn _vectors_svecf32_operator_mul(lhs: SVecf32Input<'_>, rhs: SVecf32Input<'_>) -> SVecf32Output {
+    check_matched_dims(lhs.dims() as _, rhs.dims() as _);
+
+    let size1 = lhs.len();
+    let size2 = rhs.len();
+    let mut pos1 = 0;
+    let mut pos2 = 0;
+    let mut pos = 0;
+    let mut indexes = vec![0; std::cmp::min(size1, size2)];
+    let mut values = vec![F32::zero(); std::cmp::min(size1, size2)];
+    let lhs = lhs.for_borrow();
+    let rhs = rhs.for_borrow();
+    while pos1 < size1 && pos2 < size2 {
+        let lhs_index = lhs.indexes()[pos1];
+        let rhs_index = rhs.indexes()[pos2];
+        match lhs_index.cmp(&rhs_index) {
+            std::cmp::Ordering::Less => {
+                pos1 += 1;
+            }
+            std::cmp::Ordering::Equal => {
+                let lhs_value = lhs.values()[pos1];
+                let rhs_value = rhs.values()[pos2];
+                indexes[pos] = lhs_index;
+                values[pos] = lhs_value * rhs_value;
```

Comment:
```rust
dbg!(f64::MIN_POSITIVE, f64::MIN_POSITIVE * f64::MIN_POSITIVE);
```

Reply:
It's rare but possible...

Reply:
There is similar problem in the implementation of `add` and `minus` for `svecf32`. Could you fix them by the way? @my-vegetable-has-exploded

Reply:
Sure. I didn't notice  before, but this is indeed a problem.

Reply:
`Add` and `minus` have handled this problem. https://github.com/tensorchord/pgvecto.rs/blob/09699cb48aceaa03325adca52d8e94d727fb437e/src/datatype/operators_svecf32.rs#L32